src/app/shared/components/color-picker/color-picker.component.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core'
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms'
+import { Icon, IconSize } from '../../enums/icon'
+
+const DEFAULT_COLOR: string = '#FFFFFF00'
+
+@Component({
+  selector: 'sofie-color-picker',
+  templateUrl: './color-picker.component.html',
+  styleUrls: ['./color-picker.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      multi: true,
+      useExisting: ColorPickerComponent,
+    },
+  ],
+})
+export class ColorPickerComponent implements ControlValueAccessor {
+  protected readonly Icon = Icon
+  protected readonly IconSize = IconSize
+
+  protected readonly palette: string[] = [
+    '#343434',
+    '#005919',
+    '#8d1010',
+    '#c40000',
+    '#ac29a5',
+    '#b99000',
+    '#00a99c',
+    '#265753',
+    '#370020',
+    '#1769ff',
+    '#370020',
+    '#016492',
+    '#003c58',
+    '#00ac17',
+    '#ef6c00',
+  ]
+
```

Comment:
Out of curiosity, where do these values come from? :)

Reply:
One could use css variables, but it would surprise me if that was supported in the color picker. So no good solutions at the moment – was just curious :D